src/keys/get-management-token.ts

```diff
@@ -145,7 +146,7 @@ export const getManagementToken = (privateKey: string, opts: GetManagementTokenO
   }
   return createGetManagementToken(
     createLogger({ filename: __filename }),
-    createHttpClient(),
+    createHttpClient({ prefixUrl: opts.host }),
```

Comment:
I think you need to provide a default value for it, or only set it if `host` is defined.